packages/editor/src/Theme/FontWeights.tsx

```diff
@@ -1,21 +1,22 @@
 /** @jsx jsx */
-import { jsx, useThemeUI } from 'theme-ui'
+import { jsx, useThemeUI, Theme } from 'theme-ui'
+import * as CSS from 'csstype'
 import Combobox from '../Combobox'
+import { EditorContext } from '../types'
 
 const weights = [100, 200, 300, 400, 500, 600, 700, 800, 900]
 
-export default props => {
-  const context = useThemeUI()
+const FontWeights = () => {
```

Comment:
Note: I'm changing all the `default export () => {}` to `const Foo = () => {}; default export Foo` due to https://github.com/gatsbyjs/gatsby/issues/23575.